assets/src/dashboard/app/index.js

```diff
@@ -48,7 +47,6 @@ function App({ config }) {
             <RouterProvider>
               <GlobalStyle />
               <KeyboardOnlyOutline />
-              <NavigationBar />
```

Comment:
I relocated the nav bars down to the view level so the template detail view can pass in template specific data (i.e. the title).